### DISCUSSION TITLE
Add a button to donate to the project

### DISCUSSION THREAD

**Discussion by csfalcao** [thanhlmm/refi-app#43] [Ideas]
Add a button to donate to the project. PayPal and buymeacoffee.com are easy-to-use examples.

**Comment by thanhlmm** [comment]
Hi @csfalcao 

I'm very happy that you like this project, this is the biggest donation for me! I updated the buymeacoffee link

<img width="374" alt="Screen Shot 2022-01-31 at 10 47 10" src="https://user-images.githubusercontent.com/9281080/151736151-0a229a44-b9d7-4e3c-a63f-af9104c6746e.png">

**Comment by csfalcao** [comment]
Great. Maybe adding it as a button beside "What's new" in the app lower bar can help others donate too.

**Reply by thanhlmm** [reply]
Great idea. I will add it on the next version!

**Comment by thanhlmm** [comment]
Hey @csfalcao I’m glad that you like it. I have some curious on how you use it. Do you use refi app on your own or your team? Which is the most satisfying feature and the worse?
I’m also love to hear feedback from you if you have any idea to make it better!

**Comment by csfalcao** [comment]
Hi there! First, I'm not a traditional developer, I'm a hobbyist no-code user of tools like Bubble.io and Airtable, and now I'm using FlutterFlow, a Flutter app creator. 
FlutterFlow will use only Firebase as DB and FB GUI tools are almost none. I can read some code, but I'm most dependent of GUI tools.
So, I tried Reflow, Rowy (which didn't work), and FireFoo, which is great, but the subscription plan is an overkill for me that only uses it for fun. I was hoping to find a good FB GUI app that costs ~25 USD, one time only payment and after looking for "firebase GUI app" in Google I found  RefiApp ;). I like desktop app. The features I'm looking for mostly are import/export, and basic spreadsheet tools, like copy-paste values, various cells(documents) selection and operations, sort and filters etc.

**Comment by csfalcao** [comment]
The choose you project / connect using the "I don't know how to get credential file" instant works! The commit, dif view, reverse changes all worked fine. All functions I tested worked nicely, but import.

**Reply by thanhlmm** [reply]
Great. I will check the import issue asap

**Comment by csfalcao** [comment]
Thanks
